.github/workflows/github-pages.yml

```diff
@@ -17,15 +17,46 @@ jobs:
           if [ "$project" = ".github" ]; then continue; fi
           if [ "$project" = "public" ]; then continue; fi
 
-          echo "Zipping $project"
-          zip -r ./public/$project.zip $project
-
-          echo "Building $project"
-          for projectFile in $project/*.wlp ; do
-            /usr/local/bin/entrypoint.sh WonderlandEditor --credentials "$WLE_CREDENTIALS" --windowless --package --project ./$projectFile --output ./public/
-          done
-          mv public/index.html public/$project.html
+          echo "Processing $project"
```

Comment:
I would probably prefer if each example had a `build.sh` script that we run instead, otherwise we run the simple script that was there before, i.e., glob of `*.wlp` and build. What do you think @Squareys